repositories.bzl

```diff
@@ -134,6 +134,42 @@ genrule(
             actual = "@cares_git//:ares",
         )
 
+def go_grpc_repositories(bind = True):
+    BUILD = """
+# Copyright (C) Extensible Service Proxy Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+#
+filegroup(
+    name = "test_proto",
+    srcs = ["interop/grpc_testing/test.proto"],
+    visibility = ["//visibility:public"],
+)
+"""
+    native.new_git_repository(
+        name = "org_golang_google_grpc_git",
+        commit = "9bf8ea0a8282ebecd1aa474c926e3028f5c22a4c",
+        remote = "https://github.com/grpc/grpc-go.git",
+        build_file_content = BUILD,
+    )
+
+    if bind:
```

Comment:
It seems this bind is not used.   when it is binded,  instead of depends on "@org_golang_google_grpc_git//:test_proto", you can depend on "//external:test_proto"

I saw the BUILD file use @org_golang_google_grpc_git//:test_proto directly. so bind is not needed.

Reply:
Ack. Other parts in this file uses `bind = True`. I would like to leave it to be consistent.